tests/test_app_tts_output/ttstest.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Is this file looked at at all by Caterpillar, or is its contents ignored? I don't think it has to be this complex. In fact it could be an empty or almost-empty file (we just want to test that it copies over).